_content/booklets/dhamma-in-english_suchart

```diff
@@ -0,0 +1,18 @@
+---
+title: "Dhamma in English 2014 & 2015"
+authors:
+  - suchart
+external_url: "https://www.mediafire.com/file/vmxkl1yx2mii4lu/DHAMMA_IN_ENGLISH_2016_.pdf/file"
+drive_links:
+  - "https://drive.google.com/file/d/1PUhh3lOsW-7OPR5ReZYQJU6Ck_QCzN6k/view?usp=sharing"
+course: problems
+tags:
+  - 
```

Comment:
`thai-forest` I think... 🤔 Might even move there. What do you think?

Reply:
I think we talked about this on Monday. But if not, I would keep it in problems. It fits the the topic and is hard to tag otherwise.